drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
```

Comment:
I prefer to call the resource with `<appName>_secret`
Maybe using the type of the resource in the name it's overkill because when you are looking for specific secrets you will see 'drupal_secret', but you already know that it is a secret because you are only looking secrets.

Anyways, even names from different resource types won't collide, I would append `_secret` in the resource name if it's a secret to be certainly sure the resource is a secret and to differentiate from other resources or pod name or anything called 'drupal' hehe